src/pudl_archiver/metadata/sources.py

```diff
@@ -416,4 +416,40 @@
         "license_pudl": LICENSES["cc-by-4.0"],
         "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
     },
+    "nrelss": {
+        "title": "NREL Standard Scenarios",
+        "path": "https://www.nrel.gov/analysis/standard-scenarios.html",
+        "description": (
+            "NREL's Standard Scenarios are a suite of forward-looking scenarios of the U.S."
+            "power sector that are updated annually to support and inform energy analysis."
+            "The Standard Scenarios are simulated using the Regional Energy Deployment System"
+            "and Distributed Generation Market Demand Model capacity expansion models and are"
+            "updated each year to provide timely information regarding power sector evolution."
+            "The scenarios have been designed to capture a range of possible power system"
+            "futures and consider a variety of factors from high vehicle electrification to"
+            "major cost declines for electricity generation technologies (e.g., using cost"
+            "inputs from the Annual Technology Baseline)."
+            "For select scenarios, the models are run using the PLEXOS software and the"
+            "Cambium tool that assembles structured data sets of hourly cost, emissions, and"
```

Comment:
We could consider pulling in the cambium results as well (as a second partition) but they're like 6GB each year

Reply:
If its simple to also add in the camdium results i'd say add em & add a second partition of `project` or `scenario_type` or something! but this seems like a lower priority than grabbing just the standard scenarios.

Reply:
are the camdium results 6 GB zipped? if so w/ the standard scenarios (assuming they are a similar size) its pushing up against the 50 GB archive limit

Reply:
Yeah zipped. The standard scenarios are like two orders of magnitude smaller since they don't include hourly data though, so it's less "will cambium push this archiver over the limit" and more "can we archive cambium on zenodo at all"

I'll go with "not right now" and write up cambium as a separate issue

Reply:
Tiny formatting issue -- on multi-line concatenated strings like this if we don't add an explicit space at the end of each line in the quotes, they'll run together.  Triple quoted strings are another option.